test/unit/gateways/quickpay_v10_test.rb

```diff
@@ -32,9 +32,11 @@ def test_successful_purchase
       parsed = parse(data)
       if parsed['order_id']
         assert_match %r{/payments}, endpoint
-      else
+      elsif !parsed['auto_capture'].nil?
         assert_match %r{/payments/\d+/authorize}, endpoint
-        assert_equal parsed['auto_capture'], true 
+        assert_equal parsed['auto_capture'], false 
```

Comment:
Very minor nitpick, but these should be flipped. `assert_equal expected, actual`